src/components/ModalAddressQRCode.js

```diff
@@ -36,6 +37,7 @@ class ModalAddressQRCode extends React.Component {
   }
 
   componentWillUnmount = () => {
+    $('#addressQRCodeModal').modal('hide');
```

Comment:
Fixes a bug where if the modal is open and something causes a navigation (like a reconnection), the backdrop would always be visible and the modal unclosable